store/iavlstore.go

```diff
@@ -176,7 +176,16 @@ func (st *iavlStore) Query(req abci.RequestQuery) (res abci.ResponseQuery) {
 		} else {
 			_, res.Value = tree.GetVersioned(key, height)
 		}
-
+	case "/subspace":
```

Comment:
Do we want to support limit-offset queries?

Reply:
I'm not a fan for the time being as per my upper comment - I think once we refine a way to pass in options instead of using the path, then I'm much more warm to this idea.